docs/docsite/rst/dev_guide/style_guide/index.rst

```diff
@@ -252,6 +260,59 @@ The syntax is:
    .. contents::
       :local:
 
+Accessibility guidelines
+=========================
+
+Ansible documentation has a goal to be more accessible. Use the following guidelines to help us reach this goal.
+
+Images and alternative text
+  Ensure all icons, images, diagrams, and non text elements have a meaningful alternative-text description. Do no include screen caps of CLI output. use ``code-block`` instead.
```

Comment:
```suggestion
  Ensure all icons, images, diagrams, and non text elements have a meaningful alternative-text description. Do not include screen captures of CLI output. Use ``code-block`` instead.
```
I think we want to fully say "capture" instead of "cap" to make this clearer for people who may not be familiar with this?